gems/smithy/spec/support/client_helper.rb

```diff
@@ -0,0 +1,185 @@
+# frozen_string_literal: true
+
+require 'tmpdir'
+
+module ClientHelper
+  class << self
+    def sample_shapes
```

Comment:
I think this model should work the same way our other smithy models do - we should have a model.smithy + model.json and we load that here. 
Its terrible to work in smithy json.   Its even worse to work in ruby hashes that are trying to look like json.

Reply:
The advantage to having the model json in code, prior to generation, is that we can modify contents in the test. For example v3 code has tests like:

```
        shapes['StructureShape']['members']['String']['locationName'] = 'str'
        expect(json(string: 'abc')).to eq('{"str":"abc"}')
```

Reply:
Yeah I like that ability - I guess I'm not suggesting that we can't have that - just that where the default comes from is a smithy model rather than an in code hash which is just kinda ugly and super annoying to edit.

Reply:
Ideally this is only written once, but yeah we can have a fixture default I suppose, but we need a way to modify it at runtime I think.